src/printer_yin.c

```diff
@@ -1229,10 +1226,15 @@ ypr_missing_format(struct ypr_ctx *ctx, const struct lys_module *module)
 }
 
 static void
-ypr_xmlns(struct ypr_ctx *ctx, const struct lys_module *module)
+ypr_xmlns(struct ypr_ctx *ctx, const struct lys_module *module, unsigned int indent)
 {
     unsigned int u;
-    char *space = "        ";
+    char *space;
+    if(asprintf(&space, "%*s", indent, "") == -1) {
+        LOGMEM(ctx->module->ctx);
+        ctx->out->status = LY_EMEM;
+        return;
+    } 
```

Comment:
Is this necessary? Isn't `indent + INDENT` in the `ly_print` calls sufficient (while the `space` parameter is removed)?

Reply:
Exactly, you are right.